### ISSUE TITLE
Show/hide TOC

### ISSUE THREAD

**Issue by zedzedzed** [zedzedzed/table-of-contents-plus#22]
**Reported by michael on 4 Aug 2011 01:43 UTC**
Provide a method to show/hide the TOC similar to Wikipedia.  This would use jQuery to store a cookie.

**Comment by zedzedzed** [comment]
**Comment by michael on 6 Aug 2011 13:38 UTC**
In [81]:
```CommitTicketReference repository="" revision="81"
re #24
need to figure out if this option should be enabled by default
```

**Comment by zedzedzed** [comment]
**Modified by michael on 6 Aug 2011 13:38 UTC**

**Comment by zedzedzed** [comment]
**Comment by michael on 6 Aug 2011 13:40 UTC**
It probably should be enabled by default, similar to Wikipedia.

**Comment by zedzedzed** [comment]
**Comment by michael on 7 Aug 2011 10:26 UTC**
In [82]:
```CommitTicketReference repository="" revision="82"
re #24
```

**Comment by zedzedzed** [comment]
**Comment by michael on 21 Aug 2011 04:50 UTC**
In [88]:
```CommitTicketReference repository="" revision="88"
closes #24
```

**Comment by zedzedzed** [comment]
**Modified by michael on 26 Aug 2011 10:04 UTC**
